src/proj4leaflet.js

```diff
@@ -114,6 +120,38 @@
 
 		},
 
+		// override Leaflet function?
+		latLngToPoint: function(latlng, zoom) {
+			var projectedPoint = this.projection.project(latlng);
+			var transformation = this._getTransformationByZoom(zoom);
+			var scale = this.scale(zoom);
+
+			return transformation._transform(projectedPoint, scale);
+		},
+
+		// override Leaflet function?
+		pointToLatLng: function(point, zoom) {
+			var scale = this.scale(zoom);
+			var transformation = this._getTransformationByZoom(zoom);
+			var untransformedPoint = transformation.untransform(point, scale);
+
+			return this.projection.unproject(untransformedPoint);
+		},
+
+		// override Leaflet function?
+		getProjectedBounds: function(zoom) {
+			if (this.infinite) { return null; }
+
+			var transformation = this._getTransformationByZoom(zoom);
+
+			var b = this.projection.bounds,
+				s = this.scale(zoom),
+				min = transformation.transform(b.min, s),
+				max = transformation.transform(b.max, s);
+
+			return new Bounds(min, max);
+		},
+
 		scale: function(zoom) {
 			var iZoom = Math.floor(zoom),

```

Comment:
At start of _scale_ / _zoom_ function maybe we could write something like this to avoid Leaflet function overrides:
`this.transformation = this._getTransformationByZoom(zoom);`